met-api/src/met_api/models/widget_documents.py

```diff
@@ -32,3 +33,25 @@ def get_all_by_widget_id(cls, widget_id) -> List[WidgetDocuments]:
             .filter(WidgetDocuments.widget_id == widget_id) \
             .all()
         return docs
+
+    @classmethod
+    def edit_widget_document(cls, widget_id, document_id, widget_document_data: dict) -> WidgetDocuments:
+        """Update document."""
+        widget_document = db.session.query(WidgetDocuments) \
+            .filter(WidgetDocuments.widget_id == widget_id, WidgetDocuments.id == document_id)
+        widgetdocuments: WidgetDocuments = widget_document.first()
+        if not widgetdocuments:
+            return None
+        widget_document.update(widget_document_data)
+        db.session.commit()
+        return widgetdocuments
+
+    @classmethod
+    def remove_widget_document(cls, widget_id, document_id) -> WidgetDocuments:
+        """Remove document from a document widget."""
+        deletedocument = db.session.query(WidgetDocuments) \
+            .filter(WidgetDocuments.widget_id == widget_id,
```

Comment:
write a comment about this OR usage ;ie we do the or to handle nested deletion of files within the folder